packages/ui/src/styles.css

```diff
@@ -82,24 +82,60 @@
     --canary-ai-button-stop-3: 246 66% 72%;
     --canary-ai-button-stop-4: 188 97% 89%;
     /* diff viewer */
-    --diff-grey: hsl(0, 0%, 33%);
-    --diff-add-content--: hsl(156, 28%, 10%);
-    --diff-del-content--: hsl(356, 28%, 11%);
-    --diff-add-lineNumber--: hsl(153, 40%, 14%);
-    --diff-del-lineNumber--: hsl(358, 39%, 17%);
-    --diff-plain-content--: var(--background);
-    --diff-expand-content--: var(--background);
-    --diff-plain-lineNumber--: var(--background);
-    --diff-plain-lineNumber-color--: hsl(0, 0%, 100%);
-    --diff-hunk-content--: hsl(218, 26%, 12%);
-    --diff-hunk-lineNumber--: hsl(218, 26%, 12%);
-    --diff-add-content-highlight--: hsl(152, 37%, 24%);
-    --diff-del-content-highlight--: hsl(0, 38%, 26%);
-    --diff-add-widget--: hsl(216, 80%, 65%);
-    --diff-add-widget-color--: hsl(0, 0%, 100%);
-    --diff-empty-content--: var(--background);
-    --diff-hunk-content-color--: hsl(0, 0%, 100%);
+    --diff-grey: hsl(0 0% 33%);
+    --diff-add-content--: hsl(156 28% 10%);
+    --diff-del-content--: hsl(356 28% 11%);
+    --diff-add-lineNumber--: hsl(153 40% 14%);
+    --diff-del-lineNumber--: hsl(358 39% 17%);
+    --diff-plain-content--: var(--canary-background);
+    --diff-expand-content--: var(--canary-background);
+    --diff-plain-lineNumber--: var(--canary-background);
+    --diff-plain-lineNumber-color--: hsl(0 0% 100%);
+    --diff-hunk-content--: hsl(218 26% 12%);
+    --diff-hunk-lineNumber--: hsl(218 26% 12%);
+    --diff-add-content-highlight--: hsl(152 37% 24%);
+    --diff-del-content-highlight--: hsl(0 38% 26%);
+    --diff-add-widget--: hsl(216 80% 65%);
+    --diff-add-widget-color--: hsl(0 0% 100%);
+    --diff-empty-content--: var(--canary-background);
+    --diff-hunk-content-color--: hsl(0 0% 100%);
```

Comment:
At some point we should consider removing the diff tokens from this level. They don't need to be globally accessible and should probably be defined at the component level.

Reply:
Agreed 👍 , we can scope these to the diff file. May be like a separate style file for diff component.